Frameworks/UIKit/XamlUtilities.h

```diff
@@ -82,3 +82,6 @@ UIView* GenerateUIKitControlFromXamlType(RTObject* xamlObject);
 // We need a type-safe way to do this with projections.  This is copied verbatim from the projections
 // code and works perfectly for this limited usage, but we don't do any type validation below.
 id CreateRtProxy(Class cls, IInspectable* iface);
+
+// apply LineBreakMode on xaml TextBock
+void ApplyLineBreakModeOnTextBlock(WXCTextBlock* textBlock, UILineBreakMode mode, int numberOfLines);
```

Comment:
>ApplyLineBreakModeOnTextBlock [](start = 5, length = 29)

not related to change, we should put these all into a XamlUtilities:: namespace so it's more clear at the call site.  XamlUtilities::ApplyLineBreakModeOnTextBlock would be more clear.